controllers/networkchaos/netem/types.go

```diff
@@ -207,13 +208,20 @@ func (r *Reconciler) cleanFinalizersAndRecover(ctx context.Context, networkchaos
 
 		err = r.recoverPod(ctx, &pod, networkchaos)
 		if err != nil {
-			return err
+			result = multierror.Append(result, err)
+			continue
 		}
 
 		networkchaos.Finalizers = utils.RemoveFromFinalizer(networkchaos.Finalizers, key)
 	}
 
-	return nil
+	if networkchaos.Annotations[common.AnnotationCleanFinalizer] == common.AnnotationCleanFinalizerForced {
+		r.Log.Info("Force cleanup all finalizers")
```

Comment:
Please add the target chaos object name/namespace/type in the log.

Reply:
Done